### ISSUE TITLE
Rate limited?

### ISSUE THREAD

**Issue by IndiumCrypto** [jlobos/instagram-web-api#188]
Before I'm starting to use this API. I want to know if this API is rate limited? Cause the private API is rate limited at 200 requests an hour.

**Comment by arjendevos** [comment]
Yes, instagram has a limit of 200 request an hour. This is also in your app or on the web application of instagram itself.
